WMF Framework/Event Platform Client/EventPlatformClientLibrary.swift

```diff
@@ -352,6 +352,12 @@ public class EPC {
         }
         #endif
 
+        // Make them available to any newly logged events before flushing buffer
+        queue.sync {
+            self.streamConfigurations = config
+            self.streamCopy = copies
```

Comment:
You'll be losing any previous values that were in `streamCopy` with this vs. how the old code worked. Seems like that's what you're wanting anyways, but just a heads up in case you expect `setStreamConfig` to be called more than once.

Reply:
That's a good point! As it is we're good since the stream configs are downloaded & processed only once, but once we figure out how to approach multiple stream configs (if at all) in the BUOD <> Apps sync we'll know if this needs to be modified to do, like, a merge operation.